public/app/features/plugins/admin/pages/PluginDetails.tsx

```diff
@@ -27,9 +28,24 @@ export default function PluginDetails({ match, queryParams }: Props): JSX.Elemen
 
   const plugin = useGetSingle(pluginId); // fetches the localplugin settings
   const { navModel, activePageId } = usePluginDetailsTabs(plugin, queryParams.page as PluginTabIds);
+  const { actions, info } = usePluginInfo(plugin);
   const { isLoading: isFetchLoading } = useFetchStatus();
   const { isLoading: isFetchDetailsLoading } = useFetchDetailsStatus();
   const styles = useStyles2(getStyles);
+  const subtitle = (
+    <div className={styles.subtitle}>
+      <div>{plugin?.description}</div>
+      {plugin?.details?.links && plugin.details.links.length > 0 && (
+        <div className={styles.links}>
+          {plugin.details.links.map((link) => (
+            <a key={link.name} href={link.url} className="external-link">
+              {link.name}
+            </a>
```

Comment:
could need a separator `|` here to make each link feel more destinct, a it currently look like one link "Website Licence" 
![image](https://user-images.githubusercontent.com/10999/200483990-7dfdc0ae-a985-4651-ab88-6adb75418f74.png)